list.txt

```diff
@@ -120,11 +160,16 @@ srimatrix.com
 sunrisesys.com
 suntechnologies.com
 svam.com
+synergisticit.com
+syrinx.com
+syrinx.com
```

Comment:
Looks like some dupes slipped through

Reply:
Opps, I think I discovered this on my other branch which has more spammers and a comparison script that I wrote since I wanted to make sure my email system wasn't missing any or had spammers that I didn't add to this repo. I'll fix this and then once I clean up the other branch I'll show it off. It's here if you're curious:

https://github.com/JasonD94/recruiter-spam/tree/new_spammers_9122021